rsconnect/main.py

```diff
@@ -278,6 +286,80 @@ def _test_shinyappsio_creds(server: api.ShinyappsServer):
         test_shinyapps_server(server)
 
 
+@cli.command(
+    short_help="Create a transitory admin user and provision it with an api key",
+    help="Creates an initial admin user to bootstrap a Connect instance. Returns the provisionend API key.",
+)
+@click.option(
+    "--server",
+    "-s",
+    envvar="CONNECT_SERVER",
+    help="The URL for the RStudio Connect server.",
+)
+@click.option(
+    "--insecure",
+    "-i",
+    envvar="CONNECT_INSECURE",
+    is_flag=True,
+    help="Disable TLS certification/host validation.",
+)
+@click.option(
+    "--cacert",
+    "-c",
+    envvar="CONNECT_CA_CERTIFICATE",
+    type=click.File(),
+    help="The path to trusted TLS CA certificates.",
+)
```

Comment:
use server_args instead

Reply:
I decided not to use server_args because I didn't want to expose --api-key or --name: 

--api-key because it isn't a valid parameter, --name because the server storage components rely on api_key for the connection.